### ISSUE TITLE
[Android] Sticker pack not added for Oppo CPH1719 Users

### ISSUE THREAD

**Issue by KingKlauz** [WhatsApp/stickers#450]
Hello!

**Specifications**  
Android OS Version: [7.1.1]
Device Manufacturer: [Oppo CPH1719 (CPH1719)]

**Stickers**
- ✔ My stickers do comply with the sticker requirements (512x512 and < 100 KB)
- ✔My tray icon does comply with the requirements (96x96 and < 50 KB)
- ✔ I am using **WebP** images for stickers.

**Problem Description**  
My users are facing an issue wherein they can add only the first sticker pack successfully, but fail to do so with subsequent packs. When attempting to add any sticker pack **other than the first pack**, the "add_pack_fail_prompt_update_whatsapp" error message appears. (See Attached).

![54279538_2342449839119919_5715595709894361088_n](https://user-images.githubusercontent.com/39686786/54742285-ee654700-4bfb-11e9-8b37-ad8648ab7632.png)

As can be gleaned from the screenshot, only the first sticker pack is added successfully, the rest presents an issue. However, this issue seems to **only happen with users using Oppo CPH1719 with android version 7.1.1**. I have not faced this issue with any other devices I have tested. Unfortunately, I do not have an Oppo CPH1719 to test and reproduce this error. So I am at a lost on what I can do to rectify this issue for people using Oppo CPH1719.

Additional information I have gotten from the users:
- Only using the official WhatsApp app (version 2.19.69)
- Tried reinstalling app/ restarting phone 
- Battery optimization is turned off

**Links**  
https://play.google.com/store/apps/details?id=com.klausology.singlishstickerapp

Please advice, and thank you for helping me out!

**Comment by simonzhexu** [comment]
Thank you for reporting, this is a great bug. We will try to reproduce. In the meantime, if you have a user that encounter this problem, please have them email WhatsApp customer support via: Settings->Help->Contact US in WhatsApp. And have the customer service forward the email to Simon Xu.

**Comment by simonzhexu** [comment]
Is your code the latest? Why the text contains leh in the screenshot?

**Comment by KingKlauz** [comment]
Hello Simon!

I was not using the latest code, but I have since updated and published with the new code. I am currently awaiting response from the user to see if the problem persists. 

Alright, I will ask users encountering the issue to forward the email to you. 

Also, apologies for the confusion for "leh". It's a singlish term (think of it as a period). My users like it when I include singlish terms in the app. I've simply modified the string in strings.xml file to include the term "leh":    
<string name="add_pack_fail_prompt_update_whatsapp">Sticker pack not added. Make sure you update to the latest version of WhatsApp leh.</string>

Thank you for helping out!
